src/sources/opentelemetry/http.rs

```diff
@@ -46,16 +49,22 @@ pub(crate) async fn run_http_server(
     tls_settings: MaybeTlsSettings,
     filters: BoxedFilter<(Response,)>,
     shutdown: ShutdownSignal,
+    keepalive_settings: KeepaliveConfig,
 ) -> crate::Result<()> {
     let listener = tls_settings.bind(&address).await?;
     let routes = filters.recover(handle_rejection);
 
     info!(message = "Building HTTP server.", address = %address);
 
     let span = Span::current();
-    let make_svc = make_service_fn(move |_conn| {
+    let make_svc = make_service_fn(move |conn: &MaybeTlsIncomingStream<TcpStream>| {
         let svc = ServiceBuilder::new()
             .layer(build_http_trace_layer(span.clone()))
+            .layer(MaxConnectionAgeLayer::new(
```

Comment:
It seems like if this was deleted, rustc would warn that the keepalive settings are unused? I was thinking about how we'd catch the layer accidentally being deleted.

Reply:
Yeah, that's the best way we'd catch accidentally deleting the layer.